docs/userGuide/components/advanced.md

```diff
@@ -71,6 +71,11 @@ Using the normal syntax, you are only able to use markdown formatting on heading
 </tip-box>
 <br>
 
+**Box Slot Options:**
+Slot name | Default class | Notes
+--- | --- | --- 
+_icon | (depends on box's `type` attribute) | Example: `<i slot="_icon" class="fas fa-times"></i>` will replace the icon with a cross sign in spite of the box's type.
```

Comment:
`<i slot="_icon" class="fas fa-times"></i>` is not how I normally use the `slot` technique. 
I was expecting to use it like `<span slot="icon" class="text-danger">:fas-home:</span>`. Would that work?

Reply:
You can use `<span slot="_icon" class="text-danger"><md>:fas-home:</md></span>`:
![image](https://user-images.githubusercontent.com/28982486/76678062-afe84980-660f-11ea-9c78-88ba80f96aa2.png)
The only differences are: using `_icon` instead of `icon`, using `<md>` to enforce the markdown parsing inside embedded HTML.

Reply:
Nice.
I suppose not easy to change to `icon` instead of `_icon`? If not, this is going to be the first time we expose a underscore prefixed name in the user guide.

Reply:
> Nice.
> I suppose not easy to change to `icon` instead of `_icon`? If not, this is going to be the first time we expose a underscore prefixed name in the user guide.

Should be pretty simple. Having an underscore does not change vue's slot behaviour in anyway.
`_xxx` is just the naming convention for slots we don't want to be exposed to the user https://github.com/MarkBind/vue-strap/pull/120#discussion_r366824610

Reply:
Let's use `icon` then, as we are now exposing it to the users?

Reply:
Also, are there any limitations as to what can be put in the slot? e.g., can I put a markbind thumnail or an image in the icon slot?

Reply:
> Let's use `icon` then, as we are now exposing it to the users?

Sure, will create corresponding vue-strap PR shortly.

> Also, are there any limitations as to what can be put in the slot? e.g., can I put a markbind thumnail or an image in the icon slot?

There shouldn't be any limitations as it's just replacing the placeholder in the template. Just the formatting could be messed up and need manual config.

Reply:
Good. Can try some reasonable use cases (e.g., using thumbnails, images, small animated gifs) as icons and give them as examples in the documentation.